src/main/java/com/aws/greengrass/shadowmanager/sync/model/Direction.java

```diff
@@ -0,0 +1,15 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package com.aws.greengrass.shadowmanager.sync.model;
+
+/**
+ * Enum to indicate the shadow sync directionality.
+ */
+public enum Direction {
+    BIDIRECTIONAL,
+    FROMDEVICEONLY,
+    FROMCLOUDONLY
```

Comment:
Few things here
1. I don't think you need the "ONLY" suffix
2. Should they have a "_" to separate words?
3. What should the directional reference be - source or the target:
FROM_DEVICE or TO_CLOUD?
FROM_CLOUD or TO_DEVICE?

When I think of the direction, I've been thinking of it in terms of where the data ends up - which is the opposite of what you have. 

Worth talking to @FThompsonAWS ?

Reply:
Can we include the source and target in the direction to make it clear exactly what it does?
* DEVICE_TO_CLOUD
* CLOUD_TO_DEVICE

These names also are future proof if we ever support syncing between additional source/target types, such as other core devices (i.e. sources and targets other than the local device and cloud).

With future proofing in mind, should BIDRECTIONAL include the source and target as well? If so, here's a few suggestions:
* BETWEEN_DEVICE_AND_CLOUD
* BIDIRECTIONAL_DEVICE_AND_CLOUD